<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_screen</class>
 <widget class="QMainWindow" name="Login_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>410</width>
      <height>510</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="container">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>400</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 61, 61);
border-radius: 15px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="background_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	
	background-color: rgb(81, 81, 81);
	color: rgb(255, 155, 255);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="login_button">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(70, 70, 70); 
	border: 2px solid rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(255, 255, 255)
}

QPushButton:hover{
	border : 2px solid rgb(45, 45, 45);
}



QPushButton:pressed{
	background-color: rgb(61, 61, 61); 
	border: none;
	border-radius: 10px;
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Login </string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="name">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>300</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

	background-color: rgb(70, 70, 70);
	border : 2px solid rgb(255, 255, 255);
	color: rgb(255, 255, 255);
}

QLineEdit:hover{
	border:  3px solid rgb(45, 45, 45) ;
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QPushButton" name="max_button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius: 10px;
	
	background-color: rgb(52, 199, 73);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="mini_button">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius: 10px;
	
	background-color: rgb(248, 187, 63);
}
QPushButton:pressed{
	border-radius: 10px;
	
	background-color: rgb(255, 209, 115);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="close_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius: 10px;
	background-color: rgb(249, 98, 93);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>300</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

	background-color: rgb(70, 70, 70);
	border : 2px solid rgb(255, 255, 255);
	color: rgb(255, 255, 255);
}

QLineEdit:hover{
	border:  3px solid rgb(45, 45, 45) ;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Password </string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>login_button</zorder>
      <zorder>name</zorder>
      <zorder>max_button</zorder>
      <zorder>mini_button</zorder>
      <zorder>close_button</zorder>
      <zorder>password</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
